test/it/uk/gov/hmrc/organisationsmatchingapi/services/CacheServiceSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package it.uk.gov.hmrc.organisationsmatchingapi.services
+
+import java.util.UUID
+import java.util.UUID.randomUUID
+
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.mockito.MockitoSugar
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import uk.gov.hmrc.organisationsmatchingapi.models.{Address, CrnMatch, CrnMatchingRequest, SaUtrMatch, SaUtrMatchingRequest}
+import uk.gov.hmrc.organisationsmatchingapi.services.CacheService
+import util.UnitSpec
+
+import scala.concurrent.Future
+
+class CacheServiceSpec extends UnitSpec with Matchers with GuiceOneAppPerSuite with MockitoSugar with ScalaFutures with BeforeAndAfterEach {
+
+
+  override def beforeEach() {
```

Comment:
Might not need to extende BeforeAndAfterEach and remove this empty override?

Reply:
I think this is needed and the implementation here is simply overriding the `out of the box` logic?

Reply:
Removed - don't think it's needed